### ISSUE TITLE
Jupyter kernel dies when LSTM is added to model

### ISSUE THREAD

**Issue by dma092** [keras-team/keras#8839]
I managed somehow to get keras working on python 2.7 using tensorflow.

My problem is that Jupyter kernel dies everytime keras starts the line adding the lstm in the code below.I tried to catch an exeption but nothing is throw there:
`from keras.layers import Dense, Embedding
from keras.layers import LSTM
from keras.models import Model
from keras.datasets import imdb

from __future__ import print_function
from keras.preprocessing import sequence
from keras.models import Sequential
from keras.layers import Dense, Embedding
from keras.layers import LSTM
from keras.models import Model
from keras.datasets import imdb
import pdb
max_features = len(vocabDic)
maxlen = 500  # cut texts after this number of words (among top max_features most common words)
batch_size = 32

train_set = sequence.pad_sequences(train_set, maxlen=maxlen)
test_set = sequence.pad_sequences(test_set, maxlen=maxlen)

print('Build model...')
model = Sequential()
model.add(Embedding(max_features+1, 128))
try:
    model.add(LSTM(128, dropout=0.2, recurrent_dropout=0.2))

except Exception,e:
    pdb.set_trace()
    print(test)

# try using different optimizers and different optimizer configs

`

**Comment by CharlesAuthier** [comment]
Did you try with a smaller number of samples?
Did the kernel show you an error message?
